lms/djangoapps/course_home_api/course_metadata/v1/views.py

```diff
@@ -93,6 +93,8 @@ def get(self, request, *args, **kwargs):
         can_load_courseware = courseware_meta.is_microfrontend_enabled_for_user()
 
         data = {
+            'end': course.end,
+            'user_has_passing_grade': courseware_meta.user_has_passing_grade,
```

Comment:
Why are we adding these in the course_metadata endpoint and not the progress endpoint? This endpoint is intended to be for information shared across most/all pages in the course home, but not information used on a specific tab.

Reply:
sure, moved these fields to the progress view